src/config/traefik_config.rs

```diff
@@ -0,0 +1,18 @@
+use once_cell::sync::Lazy;
+
+pub struct TraefikConfig {
+    pub fqdn: String,
+    pub network_name: String,
+}
+
+impl TraefikConfig {
+    pub fn new() -> Self {
+        TraefikConfig {
+            fqdn: std::env::var("POMEGRANATE_FQDN").unwrap_or(String::from("localhost")),
+            network_name: std::env::var("POMEGRANATE_DOCKER_NETWORK_NAME")
+                .expect("Missing proxy network !"),
+        }
+    }
+}
+
+pub static TRAEFIK_CONFIG: Lazy<TraefikConfig> = Lazy::new(TraefikConfig::new);
```

Comment:
Please avoid globals. This will make unit testing ugly.

Reply:
Hmm, fair point. I completely forgot about unit tests.
Since your concern is over the tests, should I just modify functions that needs it as parameters and pass the global arg through it ?

This would allow unit testing

Reply:
I don't think it would be nice to see a configuration struct being passed around everywhere.

IMO, we should initialize the configuration in the `main` function and then share it using a reference-counted var to everyone that needs it. Something like this:

```mermaid
classDiagram
    class DockerEngine {
        -Arc&lt;Config&gt; config
        -Docker docker
    }

    class Config {
        +TraefikConfig traefik_config
    }

    class TraefikConfig {
        +String fqdn
        +String network_name
    }
```

(The `Config` struct would later contain other configuration structs, such as the one for connecting to a database, if we still need that)

What do you think about this?

Reply:
I like the idea of the master config storing other sub-configs. 
I'll go ahead and implement that